### PR TITLE
feat: add cert delivery test

### PR DESCRIPTION
**Pull request by atanmarko**
# Description

Add local infra test that checks that certificate has been delivered to all nodes in the tce network

Fixes TP-815

## PR Checklist:

- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added or updated tests that comprehensively prove my change is effective or that my feature works

### CODE REVIEW

**Comment by JDawg287** [comment]
Maybe we should simplify the name of the test to something like: `test_cert_delivery`.

File: tests/utils.sh [hunk 1]
```diff
@@ -45,3 +45,12 @@ is_network_running() {
         echo 0
     fi
 }
+
+
+function graphql_get_tce_certificate () {
```
Comment (JDawg287) [inline tests/utils.sh:50]: Is this being used somewhere else?

File: tests/test_certificates_broadcasted.sh [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+function get_last_sequncer_generated_certificate () {
+    # $1 - node name
+    local last_certificate=`docker compose logs $1 | grep -e 'Submitting new certificate to the TCE network: ' | awk -F': ' '{print $3}' | tail -1` > /dev/null
+    echo $last_certificate
+}
+
+function check_certificate_delivered () {
+    # $1 - node name
+    docker compose logs $1 | grep -e "Certificate delivered $2" > /dev/null
```
Comment (JDawg287) [inline tests/test_certificates_broadcasted.sh:15]: A missing comment as to what is `$2` here?

File: tests/utils.sh (hunk 1 above)
Comment (atanmarko) [reply tests/utils.sh:50]: No. I thought I'll use it but it turns out it is easier to parse logs than to expose all graphql endpoints

**Comment by atanmarko** [comment]
> Maybe we should simplify the name of the test to something like: `test_cert_delivery`.

Actually I already did just accidentally added deleted file again

File: tests/utils.sh (hunk 1 above)
Comment (atanmarko) [reply tests/utils.sh:50]: I was just sorry to delete it :)

**Review by JDawg287 — APPROVED** [review]

**Pull request merged**